samples/streamlit-in-snowflake/sis-deploy/snowflake.yml

```diff
@@ -0,0 +1,9 @@
+definition_version: 1
```

Comment:
what is this used for? should it always be 1?

Reply:
Yes, it's always 1 currently. The idea (which is fairly common in other config files) is that if you change the spec in a non-backwards-compatible way, you can change the definition version so the user and the parser knows which one to use. I can definitely add a comment on this, though.